src/components/v5/common/ActionSidebar/consts.tsx

```diff
@@ -74,3 +53,49 @@ export const ACTION_BASE_VALIDATION_SCHEMA = object()
       ),
   })
   .defined();
+
+export const useCreateActionTypeNotification = (
```

Comment:
can this be a component? it's a hook returning JSX inside a file that should containt constants
Seeing how it's used, we can just replace this codesnip
![image](https://github.com/JoinColony/colonyCDapp/assets/23449297/297ff66c-99a9-45ed-8065-96c653137c8a)
with a component called `ActionTypeNotification` which can then just render the banner conditionally